metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -326,16 +326,107 @@ final class Doctor(
           _.element("thead")(
             _.element("tr")(
               _.element("th")(_.text("Build target"))
-                .element("th")(_.text("Scala"))
+                .element("th")(_.text("Type"))
                 .element("th")(_.text("Diagnostics"))
-                .element("th")(_.text("Goto definition"))
-                .element("th")(_.text("Completions"))
-                .element("th")(_.text("Find references"))
+                .element("th")(_.text("Interactive"))
+                .element("th")(_.text("Semanticdb"))
+                .element("th")(_.text("Debugging"))
+                .element("th")(_.text("Java support"))
                 .element("th")(_.text("Recommendation"))
             )
           ).element("tbody")(html => buildTargetRows(html, targetIds))
         )
     }
+
+    // Additional explanations
+    html.element("div")(
+      _.element("p")(
+        _.text(
+          "Diagnostics:"
+        )
+      ).element("ul")(
+        _.element("li")(
+          _.text(
+            s"${Icons.unicode.check} - connected build server showing diagnostics"
```

Comment:
```suggestion
            s"${Icons.unicode.check} - diagnostics correctly being reported by the build server"
```
🤔 thinking about this a bit, we could also include the build server and version here and no long need the extra section on top? Although it'd make it less up and front.

Reply:
I think it's good to show it up top in a prominent location and I think it would be a bit akward to include it in the `legend`